install.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+# Credits: Modified oh-my-zsh install script
+# 1. Handle windows
+# 2.
+
+COMMAND="socprint"
```

Comment:
Should this be `socprint.sh` instead?

Reply:
I prefer `socprint`

Reply:
This breaks the update routine, which assumes that [the current script is installed as `socprint.sh`](https://github.com/dlqs/SOCprint/blob/c01d27b10e4cacfe90f19b03d5d85c50eed59158/socprint.sh#L266)

Also the update message instructs users to [install the new script as `socprint.sh`](https://github.com/dlqs/SOCprint/blob/c01d27b10e4cacfe90f19b03d5d85c50eed59158/socprint.sh#L138)

Reply:
@dlqs I just renamed all `socprint.sh` to `socprint`. Thoughts?